bindings/core/src/method_handler/secret_manager.rs

```diff
@@ -19,12 +19,13 @@ pub(crate) async fn call_secret_manager_method_internal(
 ) -> Result<Response> {
     let response = match method {
         SecretManagerMethod::GenerateAddresses { options } => {
```

Comment:
I think I would also rename the method name 😬

Reply:
Ah, good point